README.md

```diff
@@ -14,11 +14,17 @@ random shipless speedrun generator for Outer Wilds
 1. To verify that it compiled correctly, open an Explorer window and navigate to `%APPDATA%\OuterWildsModManager\OWML\Mods\ZacBauermeister.OuterWildsRandomSpeedrun`. If you see OuterWildsRandomSpeedrun.dll in the directory, it compiled successfully.
 
 ## IDE setup for Unity in Windows
-1. Install **Unity 2019.4.39** from the [Unity Download Archive](https://unity.com/releases/editor/archive). You must install this specific version!
+1. Install **Unity 2019.4.39** from the [Unity Download Archive](https://unity.com/releases/editor/archive). **You must install this specific version!** Select the Unity Hub from the "Downloads (Win)" dropdown.
+    1. When running the installer, make sure to deselect "Microsoft Visual Studio Community 2019".

```

Comment:
Are these steps specifically for VSCode? Does Installing VS cause problems?

Reply:
Documenting a verbal discussion: @bucketon to handle the VS setup with a separate section in a follow-up PR.